Fix Area node description mistake

jaymehta-g opened the pull request:
<!--
Please target the `master` branch in priority.
PRs can target `3.x` if the same change was done in `master`, or is not relevant there.

Relevant fixes are cherry-picked for stable branches as needed by maintainers.
You can mention in the description if the change is compatible with `3.x`.
-->
Change brief descriptions for Area2D and Area nodes to be less confusing.

For Area2D:
Changed
"2D area for detection and physics and audio influence."
to
"2D area for detection, physics, and audio influence."

For Area (Area3D):
"3D area for detection and physics and audio influence."
to
"3D area for detection, physics, and audio influence."

jaymehta-g committed:
Changed Area2D node description

Changed description to be less confusing

Changed
"2D area for detection and physics and audio influence."
To
"2D area for detection, physics, and audio influence."

jaymehta-g committed:
Changed Area3D node description

Changed description to be less confusing

Changed
"3D area for detection and physics and audio influence."
To
"3D area for detection, physics, and audio influence."

timothyqiu commented:
As expanded in the description, the original brief description means an area can:

* detect (other body and area)
* influence physics and audio

I think this modification changes the meaning, making "detection", "physics", and "audio influence" the same level.

souplamp reviewed (changes requested):
"2D/3D area for collision detection, as well as physics and audio influence."
Perhaps this wording would help emphasize the difference?

skyace65 reviewed (changes requested):
souplamp's suggested change would be perfect.

jaymehta-g committed:
Revise Area node brief descriptions

Revision to make the separation between collision and physics/audio influence more clear

jaymehta-g commented:
Just changed it to better reflect the separation between collision and physics/audio

I still don't know if it's accurate to call it "collision detection" since no physics collision actually happens with an Area as far as I know, should it be instead called a "collider detection"?

souplamp reviewed (approved):
> I still don't know if it's accurate to call it "collision detection" since no physics collision actually happens with an Area as far as I know, should it be instead called a "collider detection"?

Areas inherit from CollisionObject2D/3D, and they need CollisionShapes/Polygons, but I can see why that phrasing could still be confusing (implying it interacts with physics when that's not necessarily true). I think you could leave it as simply "detection" just to keep it concise, and users can read the description below for more information.

skyace65 commented:
I agree with souplamp again. Leaving it as "detection" is probably for the best

YuriSizov commented:
Bump :)

jaymehta-g commented:
> Bump :)

forgot about this lmao. ill change it to just "detection" and commit again

jaymehta-g committed:
Second revision of area node desc

skyace65 commented:
Thanks! And congrats on your first merged PR!
